_security-plugin/access-control/api.md

```diff
@@ -2,7 +2,7 @@
 layout: default
 title: API
 parent: Access control
-nav_order: 90
+nav_order: 120
```

Comment:
Generic question: Changing `nav_order` will re-organize this document based on the value entered. Correct?

Reply:
oo nice..thank you for sharing the final view. This helps a lot in visualizing the "re-organization" of the Security Plugin  section

Reply:
Agreed. It helps.